### PR TITLE
chore(docs): update docs to use webdriver-manager for mobile setup

### PR DESCRIPTION
**Pull request by sjelin**

### CODE REVIEW

File: docs/mobile-setup.md [hunk 1]
```diff
@@ -7,65 +7,9 @@ does not yet officially support or run its own tests against a particular config
...
-   * Once you download the apk, install to your AVD as such:
-
-```shell
-> adb install ~/Desktop/chrome-browser-google-34.0.1847.114-www.apk4fun.com.apk 
-2323 KB/s (30024100 bytes in 12.617s)
-Success
-```
-* If you check your AVD now, it should have Chrome.
+Use `webdriver-manager` to install `appium` and the Android SDK.  See details
+[here](https://github.com/angular/webdriver-manager/blob/master/mobile.md).
```
Comment (juliemr) [inline docs/mobile-setup.md]: Documentation nit: prefer useful descriptions in links, e.g. `See details [on the WebDriverManager page]` (a Tech Writer drilled this into my head a couple years ago so now I have to comment on it).

File: docs/mobile-setup.md [hunk 2]
```diff
@@ -74,20 +18,13 @@ Success
 > npm start # or `./scripts/web-server.js`
 Starting express web server in /workspace/protractor/testapp on port 8000
 ```
-*   If your AVD isn't already started from the setup, start it now:
 
-```shell
-> emulator -avd LatestAndroid
-```
-*   Start Appium:
+*   Start appium and the Android Emulators (details [here](
```
Comment (juliemr) [inline docs/mobile-setup.md]: Ditto, replace [here] with [on the Webdriver Manager docs]

File: docs/mobile-setup.md [hunk 3]
```diff
@@ -106,17 +43,7 @@ exports.config = {
...
-  // configuring wd in onPrepare
-  // wdBridge helps to bridge wd driver with other selenium clients
-  // See https://github.com/sebv/wd-bridge/blob/master/README.md
-  onPrepare: function () {
-    var wd = require('wd'),
-      protractor = require('protractor'),
-      wdBridge = require('wd-bridge')(protractor, wd);
-    wdBridge.initFromProtractor(exports.config);
-  }
+  baseUrl: 'http://10.0.2.2:8000'
```
Comment (juliemr) [inline docs/mobile-setup.md]: You would still need to use `wd` or `webdriver-js-extender`, right?

Comment (sjelin) [inline docs/mobile-setup.md]: yeah, let's just block this on https://github.com/angular/protractor/pull/3326.  `wd-bridge` was super obnoxious to use

**Comment by sjelin** [comment]
Removing the stuff about getting rid of `wd-bridge` until `webdriver-js-extender` is in a better place.  All comments addressed

**Comment by juliemr** [comment]
LGTM, travis errors are sauce timeouts and not related to documentation changes.

**Pull request merged**